README.md

```diff
@@ -4,53 +4,57 @@
 
 
 
-##### What I do?
+#### What I do?
 
-I aim to give users a new way to interact with Google Maps through engaging text-based conversational interfaces. 
+I aim to give users a new way to interact with Google Maps through engaging text-based conversational interfaces.
 
-##### How old am I?
+#### How old am I?
 
 I'm only a baby bot right now, I need you to feed me with logic, data and inspiration.
 
-##### What is the motivation behind building me?
+#### What is the motivation behind building me?
 
-The primary motivation of the developers of MapBot is to provide a playground to tech enthusiasts, both beginners and advanced to try algorithms, approaches and ideas while contributing to a real-life project. 
+The primary motivation of the developers of MapBot is to provide a playground to tech enthusiasts, both beginners and advanced to try algorithms, approaches and ideas while contributing to a real-life project.
 
-##### What I aspire to be one day?
+#### What I aspire to be one day?
 
 - I want to help users in the most comprehensive way.
 - I want to give 'geeks' a platform to try out all things 'cool'.
 
 ------
 
-##### Are you here for GSSoC 2020?
+#### Are you here for GSSoC 2020?
 
 Check out all related information [here](GSSoC.md)
 
 ------
 
-##### What are some pre-requisites?
+#### What are some pre-requisites?
 
--  MySQL 
-  - Install the community version of mySQL from the [official mySQL documentation page](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/). 
+- MySQL
+  - Install the community version of mySQL from the [official mySQL documentation page](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/).
   - Create root user credentials during installation.
   - Verify the installation, running the command  `mysql -uroot -p -hlocalhost` should open the mySQL monitor. (Enter the root password when prompted)
 - StanfordCoreNLP
-  - StanfordCoreNLP has a dependency on Java 8. `java -version` should complete successfully with version 1.8 or higher. 
+  - StanfordCoreNLP has a dependency on Java 8. `java -version` should complete successfully with version 1.8 or higher.
   - Windows- Download as a .zip file from [here](https://stanfordnlp.github.io/CoreNLP/download.html).  
   - Linux and MacOS- Follow the instructions to download the file from [here](https://stanfordnlp.github.io/CoreNLP/download.html).  
+- Docker (if you want to set me up the Docker way)
```

Comment:
How about adding the Docker setup pre-requisites together with the Docker setup instructions, I do not want to clutter the original space, I often find developers struggling during first time setup. This would be a nightmare to explain.
Also add the StanfordCoreNLP point to the pre-requisites too. We will remove that soon but for the time being. (Use the same statement in the original readme)

Reply:
Made different pre-requisites.